includes/definitions/toshiba.yaml

```diff
@@ -0,0 +1,14 @@
+os: toshiba
```

Comment:
I think we should change this to toshiba-printer, toshiba is too generic for a printer.